client/src/components/Header/SearchBar/SearchBar.js

```diff
@@ -45,17 +51,19 @@ export default class SearchBar extends React.Component {
             d: 25,
             address: locationAddress
         });
-        window.location.replace(`/search/?${queryString}`);
+        window.location.href = `/search/?${queryString}`;
```

Comment:
Is there a reason this was changed?

Reply:
`replace` doesn't add a page to the history, so pressing back didn't work right